docs/getting-started/installation/wpilib.md

```diff
@@ -0,0 +1,6 @@
+# Installing WPILib
+Follow the WPILib Docs [instructions for installing WPILib](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/wpilib-setup.html#wpilib-installation-guide). When prompted to install VSCode, you may choose 'Skip and don't use VS Code', though it is recommended to install it to be used when updating 3rd party vendor dependencies.
+
+To install WPILib you can follow the [guide](https://www.jetbrains.com/help/idea/installation-guide.html) contained on the WPILib wiki, with a few special instructions. Once you reach the page where it asks which install mode you'd like, make sure to select "Everything", and then on the next page select "Skip and don't use VS Code".
```

Comment:
```suggestion

```

Reply:
I'm not sure why it keeps doing this, I think its something with automatically merging if I forget to pull before writing my stuff?